spec/controllers/departments_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe DepartmentsController do
+  
+  before(:each) { login_admin }
+
+  describe "GET orgchart" do
+    let!(:department) { create(:department) }
+    
+    # catch view rendering errors
+    render_views
+    
+    it "shows an orgchart" do
+      get :orgchart, {id: department.id}
+      expect(response).to be_success
+    end
+    
+    context "as a reader" do
+      before(:each) { login_as('Reader') }
```

Comment:
I wasn't sure which roles should have access to the department orgcharts so I arbitrarily picked `Reader` as a role without permission, for illustrative purposes.